docs/guides/postgres/reconfigure-tls/reconfigure-tls.md

```diff
@@ -0,0 +1,710 @@
+---
+title: Reconfigure Postgres TLS/SSL Encryption
+menu:
+  docs_{{ .version }}:
+    identifier: pg-reconfigure-tls-cluster
+    name: Reconfigure Postgres TLS/SSL Encryption
+    parent: pg-reconfigure-tls
+    weight: 10
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
+
+# Reconfigure Postgres TLS/SSL (Transport Encryption)
+
+KubeDB supports reconfigure i.e. add, remove, update and rotation of TLS/SSL certificates, changing issuer for existing Postgres database via a PostgresOpsRequest. This tutorial will show you how to use KubeDB to reconfigure TLS/SSL encryption.
+
+## Before You Begin
+
+- At first, you need to have a Kubernetes cluster, and the kubectl command-line tool must be configured to communicate with your cluster. If you do not already have a cluster, you can create one by using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+- Install [`cert-manger`](https://cert-manager.io/docs/installation/) v1.0.0 or later to your cluster to manage your SSL/TLS certificates.
```

Comment:
add cert-manager installation link as previous format, for example "Install cert-manager in your cluster at `here`. add link here.